javers-core/src/main/java/org/javers/core/graph/EdgeBuilder.java

```diff
@@ -53,7 +54,7 @@ MultiEdge createMultiEdge(Property containerProperty, EnumerableType enumerableT
         Object container = node.getPropertyValue(containerProperty);
 
         EnumerableFunction edgeBuilder = null;
-        if (enumerableType instanceof MapType){
+        if (enumerableType instanceof MapType || enumerableType instanceof MultimapType){
```

Comment:
znowu, jeśli zrobisz, że `MultimapTyp extends MapType` to nic nie trzeba tu zmieniać